packages/utils/src/cache/storages/indexeddb.ts

```diff
@@ -0,0 +1,177 @@
+import { openDB } from 'idb';
+import type { DBSchema, IDBPDatabase } from 'idb';
+import type { CacheMap } from '../types';
+import type { CacheItem } from '../interfaces/CacheItem';
+import type { CacheStorage } from '../interfaces/CacheStorage';
+import { StoragePrefix } from '../constants.js';
+import { getItemKey } from '../helpers.js';
+
+interface IndexedDBDatabase extends DBSchema {
+  [key: StoreName]: {
+    key: string;
+    value: CacheItem;
+  }
+}
+
+/**
+ * Literal type for dynamic store name casting to idb
+ */
+type StoreName = `[${StoragePrefix.DEFAULT}][${string}]`;
```

Comment:
Why `StoreName` ? I think it would sound nicer if it aligns with `dbName`

Reply:
Updated by following your suggestion.